randline.py

```diff
@@ -1,10 +1,40 @@
 import random
 import sys
+import datetime
+
+def timeOfDay(time):
+    if time.hour <= 2 or time.hour >= 6:
+        return 'evening'
```

Comment:
Because the time library returns time using a 24 hour clock, this 6 would be 6am, which is not evening. I expect you meant 6pm or 18 on a 24 hr clock.